src/scene/text/AbstractText.ts

```diff
@@ -478,124 +476,6 @@ export abstract class AbstractText<
         this.onViewUpdate();
     }
 
-    /**
```

Comment:
I'm not sure I understand why you're removing width, height, getSize and setSize from here. Can you clarify?